app.js

```diff
@@ -104,19 +92,38 @@ class ToDoList {
         for (let i = 0; i < taskList.length; i++) {
```

Comment:
Wcześniej na to nie zwróciłem uwagi, ale to co tu robisz jest trochę hmm 'przerośnięte'. Użyj funkcji `.find` ktora znajduje w arrayce konkretny element.